tests/test_modules/valueset_bundle.json

```diff
@@ -0,0 +1,155 @@
+{
+  "resourceType": "Bundle",
+  "id": "81e8eeb8-573b-4b6c-b3e3-1cc6e6dc9e48",
+  "meta": {
+    "lastUpdated": "2021-11-04T23:28:32.722+00:00"
+  },
+  "type": "searchset",
+  "total": 1,
+  "link": [ {
+    "relation": "self",
+    "url": "http://fhir.pb.dev.cnics.cirg.washington.edu/fhir/ValueSet?url=http%3A%2F%2Fcnics-cirg.washington.edu%2Ffhir%2FValueSet%2FCNICS-COPD-codings"
```

Comment:
```suggestion
    "url": "http://fhir.pb.dev.cnics.cirg.washington.edu/fhir/ValueSet?url=http://cnics-cirg.washington.edu/fhir/ValueSet/CNICS-COPD-codings"
```

Reply:
Those file contents come directly from a HAPI query, and IMO, match the HTTP spec, needing to URL encode query string parameters.  You sure you want that change?

Reply:
Good points, thanks!